docs/DeveloperGuide.md

```diff
@@ -366,11 +366,45 @@ Classes added for this feature:
 - `HighImportanceCommand`
 - `HighImportanceParser`
 
+Given below is an example scenario of the high importance flag feature.
+
+Step 1. When a user is created, they are automatically assigned `NOT_HIGH_IMPORTANCE` as their importance status.
+
+![ImportanceState0](images/high-importance-flag/ImportanceState0.png)
+
+Step 2. The user executes `impt 1` command to change the high importance status of the 1st person in the address book. The `impt` command executes and determines the current `highImportanceStatus` of the contact (which in this case is `NOT_FAVOURITE`). It then calls the private helper method `createFavouritedPerson` to create a copy of the target contact, except with their `highImportanceStatus` now pointing to the opposite `highImportanceStatus` value (which will be `HIGH_IMPORTANCE`).
```

Comment:
I think you meant `NOT_HIGH_IMPORTANCE` and `createHighImportancePerson`? I might be wrong! 😅

Reply:
Thanks for spotting the error, keen eyes! 👍